### PR TITLE
add ticket message creation endpoints

### PR DESCRIPTION
**Pull request by joaoinacio**

### CODE REVIEW

**Review by tiagobastos — DISMISSED** [review]

**Review by BrechtBonte — DISMISSED** [review]

File: apiary.apib [hunk 1]
```diff
@@ -8347,6 +8350,67 @@ Gets the ticket message.
...
 
+
+### tickets.importMessage [POST /tickets.importMessage]
+
+Imports an existing message to a ticket.
+
++ Request (application/json)
+    + Attributes (object)
+        + id: `f29abf48-337d-44b4-aad4-585f5277a456` (string, required)
+        + body: `<p>Hello world</p>` (string) - Uses HTML formatting
```
Comment (Jdww) [inline apiary.apib]: This is required

File: apiary.apib [hunk 2]
```diff
@@ -8347,6 +8350,67 @@ Gets the ticket message.
...
+        + sent_by (object)
```
Comment (Jdww) [inline apiary.apib]: This is required

File: apiary.apib [hunk 3]
```diff
@@ -8347,6 +8350,67 @@ Gets the ticket message.
...
+            + `type`: `contact` (enum)
```
Comment (Jdww) [inline apiary.apib]: this is required

File: apiary.apib [hunk 4]
```diff
@@ -8347,6 +8350,67 @@ Gets the ticket message.
...
+
+
+### tickets.addReply [POST /tickets.addReply]
+
+Adds a message to a ticket.
+
++ Request (application/json)
+    + Attributes (object)
+        + id: `f29abf48-337d-44b4-aad4-585f5277a456` (string, required)
+        + body: `<p>Hello world</p>` (string) - Uses HTML formatting
```
Comment (Jdww) [inline apiary.apib]: This is required

File: apiary.apib [hunk 5]
```diff
@@ -8347,6 +8350,67 @@ Gets the ticket message.
...
+
+
+### tickets.addInternalMessage [POST /tickets.addInternalMessage]
+
+Adds an internal message to a ticket.
+
++ Request (application/json)
+    + Attributes (object)
+        + id: `f29abf48-337d-44b4-aad4-585f5277a456` (string, required)
+        + body: `<p>Hello world</p>` (string) - Uses HTML formatting
```
Comment (Jdww) [inline apiary.apib]: This is required

**Review by tiagobastos — APPROVED** [review]

**Review by Jdww — APPROVED** [review]

**Comment by joaoinacio** [comment]
✅

**Review by gdegirmenci — APPROVED** [review]

**Pull request merged**